compiler/arser/include/arser/arser.h

```diff
@@ -737,6 +728,34 @@ template <typename T> T Arser::get(const std::string &arg_name)
   return get_impl(arg_name, static_cast<T *>(nullptr));
 }
 
+class Helper
+{
+public:
+  static void add_version(Arser &arser, const std::function<void(void)> &func)
+  {
+    arser.add_argument("--version")
+      .nargs(0)
+      .required(false)
+      .default_value(false)
+      .help("Show version information and exit")
+      .exit_with(func);
+  }
+
+  static void add_verbose(Arser &arser)
+  {
+    arser.add_argument("-V", "--verbose")
+      .nargs(0)
+      .required(false)
+      .default_value(false)
+      .help("output additional information to stdout or stderr");
+  }
+
+  static void add_string(Arser &arser, const std::string &name, const std::string &help)
+  {
+    arser.add_argument(name).nargs(1).type(arser::DataType::STR).help(help);
+  }
+};
```

Comment:
I think `add_version` and `add_verbose` are very good methods to have. But `add_string` might a bit confuse a reader because of its name and `add_argument` is kinda intuitive expression to me. 

How about just omit `nargs` and `type` method? As of now, `nargs` has default value "1" and `type` can have `arser::DataType::STR` default value with a single PR. Then,
```cxx
arser.add_argument("--by_name")
    .nargs(1)
    .type(arser::DataType::STR)
    .help("Input operation name to select nodes.");
```
could be
```cxx
arser.add_argument("--by_name")
    .help("Input operation name to select nodes.");
```

Reply:
> type can have arser::DataType::STR default value with a single PR.

Can you post this as a PR?

Reply:
@seanshpark Sure:)